draft-ietf-quic-transport.md

```diff
@@ -2858,8 +2858,8 @@ discussed in more detail in {{QUIC-RECOVERY}}.
 
 To limit ACK Blocks to those that have not yet been received by the sender, the
```

Comment:
To me it seems that the goal stated here is inaccurate. The goal is to limit the size of ACK frames.

So maybe something like: _And endpoint SHOULD minimize the size of acknowledgements. It SHOULD track which ACK frames have been acknowledged by its peer, and once an ACK frame is acknowledged, refrain from re-acknowledging the packets._